recipes/libradolan/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+if [ "$(uname)" == "Darwin" ]
+then
+    # for Mac OSX
+    export CC=clang
+    export CXX=clang++
+    export MACOSX_VERSION_MIN="10.7"
+    export MACOSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
```

Comment:
Unfortunately, that won't work. `-stdlib=libstdc++` forces one to use `gcc`, which on Mac is 4.2.1.